CMakeLists.txt

```diff
@@ -28,11 +28,21 @@ SET(TARGET "aspect")
 FILE(GLOB_RECURSE TARGET_SRC
      "source/*.cc" "unit_tests/*.cc" "include/*.h" "contrib/catch/catch.hpp")
 
-# For the unity build we need to compile helper_functions.cc before core.cc
-# ensure this by reordering the files
-FIND_FILE(HELPER_PATH helper_functions.cc HINTS "${CMAKE_SOURCE_DIR}/source/simulator")
-LIST(REMOVE_ITEM TARGET_SRC ${HELPER_PATH})
-LIST(INSERT TARGET_SRC 0 ${HELPER_PATH})
+# special treatment of some files for unity builds:
+LIST(APPEND UNITY_LAST_FILES
+	"source/simulator/core.cc;source/volume_of_fluid/handler.cc")
```

Comment:
so helper_functions.cc is no longer needed?

Reply:
helper_functions was moved to the front to be in front of core.cc, Now I move things to the back instead.